tests/serverpod_test_server/test_e2e/database_roundtrip_test.dart

```diff
@@ -0,0 +1,259 @@
+import 'package:serverpod_test_client/serverpod_test_client.dart';
+import 'package:serverpod_test_server/test_util/config.dart';
+import 'package:test/test.dart';
+
+Future<void> setupTestData(Client client) async {
+  await client.basicDatabase.deleteAllSimpleTestData();
+  await client.basicDatabase.createSimpleTestData(100);
+}
+
+void main() {
+  var client = Client(serverUrl);
+
+  group('Database', () {
```

Comment:
Suggestion: Could you split out each type into individual tests, to align with the general test style. 

This will also be helpful when things break as we can see directly what is going wrong without having to inspect and understand these convoluted tests. 

Rename this file to include the word "type" like, `database_roundtrip_type_test.dart` or similar. 

Anything that is not a type test shouldn't be located in this file.

Reply:
Broke out all the individual cases into distinct test files.